src/node_perf.cc

```diff
@@ -366,6 +366,20 @@ void Timerify(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(wrap);
 }
 
+// Notify a custom PerformanceEntry to observers
+void Notify(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Utf8Value type(env->isolate(), args[0]);
+  Local<Value> entry = args[1];
+  PerformanceEntryType entry_type = ToPerformanceEntryTypeEnum(*type);
+  AliasedUint32Array& observers = env->performance_state()->observers;
+  if (entry_type != NODE_PERFORMANCE_ENTRY_TYPE_INVALID &&
+      observers[entry_type]) {
+    env->performance_entry_callback()->Call(env->context(), Undefined(env->isolate()), 1, &entry);
```

Comment:
I think this line exceeds 80 characters, and you may want to wrap this with `USE(...);` since the return value is unused and this might otherwise emit a compiler warning? (Not using the return value is the right thing to do here, though.)

Reply:
done